src/setup.js

```diff
@@ -1,40 +1,55 @@
 const mintAndDeposit = require('../tests/actions/mintAndDeposit');
 
-module.exports = async function(contracts, nodes, accounts, web3) {
+module.exports = async function(contracts, nodes, accounts, wallet) {
   const alice = accounts[0].addr;
 
-  let data = contracts.token.methods.addMinter(alice).encodeABI();
-  await contracts.governance.methods.propose(contracts.token.options.address, data).send({
-    from: alice, gas: 2000000
-  });
-  await contracts.governance.methods.finalize().send({ from: alice });
-  await contracts.token.methods.mint(alice, 500000000000).send({from: alice});
-  await contracts.token.methods.approve(contracts.exitHandler.options.address, 500000000000).send({from: alice});
-  await contracts.token.methods.approve(contracts.operator.options.address, 500000000000).send({from: alice});
+  let data = contracts.token.interface.functions.addMinter.encode([alice]);
+  let tx = await contracts.governance.propose(contracts.token.address, data, { gasLimit: 2000000 });
+  await tx.wait();
```

Comment:
one-liner if we don't need `tx` later on: https://github.com/leapdao/leap-guardian/blob/master/scripts/setValidator.js#L61